src/OpenTelemetry/Trace/ActivityExportProcessorOptions.cs

```diff
@@ -0,0 +1,25 @@
+// <copyright file="ActivityExportProcessorOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Trace
+{
+    public class ActivityExportProcessorOptions
```

Comment:
We should have `IConfiguration` overloads here IIRC. Here's the version I did in main-logs: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main-logs/src/OpenTelemetry/Trace/ExportActivityProcessorOptions.cs

Reply:
Agree, though my plan for this PR was to just use it to discuss the approach of not trying to solve all our issues with OtlpExporterOptions. I had planned to close it. I figured a lot of this would be integrated in from main-logs.

Though, since you're integrating main-logs in iteratively, I'm also open to continuing forward with this PR once the necessary bits from the SDK are in place and made public.